utils/tiles.py

```diff
@@ -19,7 +23,7 @@ def __init__(self, min_zoom, max_zoom):
         self.min_zoom = min_zoom
         self.max_zoom = max_zoom
 
-    def filter_zooms_in_range(self, zooms):
+    def filter_zooms_in_range(self, zooms=None):
         # type: (List[int]) -> List[int]
         """ given a list of zooms return zooms within min_zoom and max_zoom"""
         if bool(zooms):
```

Comment:
```suggestion
        if zooms is None:
```

Reply:
just in case some people might pass [] so bool() here will handle both [] and None case.

Reply:
Ah, then you can just say `if zooms:`